MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/dds/ImageClusteringActivity.java

```diff
@@ -65,12 +67,17 @@ protected void onCreate(Bundle savedInstanceState) {
   }
 
   @Override
-  public void onMapReady(MapboxMap mapboxMap) {
+  public void onMapReady(@NonNull MapboxMap mapboxMap) {
     this.mapboxMap = mapboxMap;
-    initLayerIcons();
-    addClusteredGeoJsonSource();
-    Toast.makeText(ImageClusteringActivity.this, R.string.zoom_map_in_and_out_instruction,
-      Toast.LENGTH_SHORT).show();
+    mapboxMap.setStyle(Style.LIGHT, new Style.OnStyleLoaded() {
+      @Override
+      public void onStyleLoaded(@NonNull Style style) {
+        initLayerIcons();
```

Comment:
add style as parameter to initLayerIcons

Reply:
fixed